tests/CppUTestExt/MockFailureTest.cpp

```diff
@@ -205,9 +205,9 @@ TEST(MockFailureTest, MockExpectedParameterDidntHappenFailure)
     MockExpectedParameterDidntHappenFailure failure(UtestShell::getCurrent(), "foo", *list);
     STRCMP_EQUAL("Mock Failure: Expected parameter for function \"foo\" did not happen.\n"
                  "\tEXPECTED calls that DID NOT happen related to function: foo\n"
-                 "\t\tfoo -> int bar: <2>, const char* boo: <str>\n"
+                 "\t\tfoo -> int bar: <2 (0x00000002)>, const char* boo: <str>\n"
                  "\tACTUAL calls that DID happen related to function: foo\n"
-                 "\t\tfoo -> int bar: <10>, const char* boo: <bleh>\n"
+                 "\t\tfoo -> int bar: <10 (0x0000000a)>, const char* boo: <bleh>\n"
```

Comment:
Yes, please stick with 0xa. You can get into untold trouble doing 0x0000000a on different bit width systems....... 0xa meaning, only non-zero nibbles are displayed. Because it can be really tough to figure out now many 0 there should be and then testing that on different platforms (bad enough we have that problem with BITS_EQUAL.....)